src/styles/blocks/catalog.scss

```
@@ -17,9 +19,16 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: transform $smooth;
```

As you change color on hover too, so needs to add color to transition as well
```suggestion
    transition: transform $smooth, color $smooth;
```